geoportailv3/static/js/query/querydirective.js

```diff
@@ -21,7 +21,8 @@ app.queryDirective = function(appQueryTemplateUrl) {
     scope: {
       'map': '=appQueryMap',
       'infoOpen': '=appQueryOpen',
-      'appSelector': '=appQueryAppselector'
+      'appSelector': '=appQueryAppselector',
+      'measureOpen': '=appQueryMeasureopen'
```

Comment:
To me the query directive should not know about the measure component. Why not use `appQueryActive` as we do in other directives?